xm-commons-lep/src/main/java/com/icthh/xm/commons/lep/XmLepLoggingExecutorListener.java

```diff
@@ -35,9 +49,23 @@ public void accept(final LepExecutorEvent lepExecutorEvent) {
 
     private void onBeforeEvent(BeforeResourceExecutionEvent beforeEvent) {
 
-        log.info("lep:start: execute lep at [{}], script: {}",
-                 buildLepSignature(beforeEvent.getMethod()),
-                 beforeEvent.getKey());
+        LepLogConfiguration loggingConfig = loggingConfigService.getLepLoggingConfig(beforeEvent.getKey().getId());
+
+        if (Objects.isNull(loggingConfig)) {
```

Comment:
please use operator == for null comparison inIF statements. Objects.isNull s for functional style.
Please change all places

Reply:
changed